File: scripts/generate-plan-crecimiento-pdf.mjs
import { chromium } from 'playwright';
import { mkdirSync } from 'fs';
import { dirname, join } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
const root = join(__dirname, '..');
const outDir = join(root, 'marketing');
mkdirSync(outDir, { recursive: true });

const outPath = join(outDir, 'plan-crecimiento-instagram.pdf');

const html = `<!DOCTYPE html>
<html lang="es">
<head>
<meta charset="utf-8" />
<title>Plan de crecimiento Instagram — Informática González</title>
<style>
  @page { size: A4; margin: 18mm 16mm; }
  * { box-sizing: border-box; }
  body {
    font-family: "Segoe UI", system-ui, sans-serif;
    color: #0a0a0a;
    line-height: 1.45;
    font-size: 11.5px;
    margin: 0;
  }
  h1 {
    font-size: 22px;
    margin: 0 0 6px;
    letter-spacing: -0.02em;
  }
  h2 {
    font-size: 13px;
    text-transform: uppercase;
    letter-spacing: 0.12em;
    color: #2563eb;
    margin: 22px 0 10px;
    padding-bottom: 4px;
    border-bottom: 1px solid #e5e5e5;
  }
  h3 {
    font-size: 12px;
    margin: 14px 0 6px;
    color: #171717;
  }
  p { margin: 0 0 8px; color: #404040; }
  .muted { color: #737373; font-size: 10.5px; }
  .hero {
    background: #0a0a0a;
    color: #fcfcfc;
    padding: 22px 24px;
    margin: 0 0 18px;
  }
  .hero p { color: #a3a3a3; margin: 0; }
  .hero strong { color: #fcfcfc; }
  .accent { color: #2563eb; }
  .badge {
    display: inline-block;
    background: #2563eb;
    color: white;
    font-size: 9px;
    letter-spacing: 0.14em;
    text-transform: uppercase;
    padding: 4px 8px;
    margin-bottom: 10px;
  }
  table {
    width: 100%;
    border-collapse: collapse;
    margin: 8px 0 12px;
    font-size: 10.5px;
  }
  th, td {
    border: 1px solid #e5e5e5;
    padding: 7px 8px;
    text-align: left;
    vertical-align: top;
  }
  th {
    background: #f5f5f5;
    font-weight: 600;
    font-size: 10px;
    text-transform: uppercase;
    letter-spacing: 0.06em;
    color: #525252;
  }
  ul { margin: 6px 0 10px; padding-left: 18px; color: #404040; }
  li { margin-bottom: 4px; }
  .box {
    border: 1px solid #e5e5e5;
    padding: 12px 14px;
    margin: 10px 0;
    background: #fcfcfc;
  }
  .box strong { color: #0a0a0a; }
  .phases {
    display: grid;
    grid-template-columns: 1fr 1fr 1fr;
    gap: 10px;
    margin: 10px 0;
  }
  .phase {
    border: 1px solid #e5e5e5;
    padding: 12px;
    min-height: 140px;
  }
  .phase .pct {
    font-size: 20px;
    font-weight: 700;
    color: #2563eb;
    margin: 0 0 4px;
  }
  .phase h3 { margin-top: 0; }
  .footer {
    margin-top: 28px;
    padding-top: 10px;
    border-top: 1px solid #e5e5e5;
    font-size: 10px;
    color: #737373;
    display: flex;
    justify-content: space-between;
  }
  .summary {
    background: #eff6ff;
    border-left: 3px solid #2563eb;
    padding: 10px 12px;
    margin: 12px 0;
  }
</style>
</head>
<body>

<div class="hero">
  <div class="badge">Marketing · Instagram</div>
  <h1>Plan de crecimiento de seguidores</h1>
  <p><strong>Informática González</strong> · Estrategia orgánica + Meta Ads</p>
  <p class="muted" style="color:#737373;margin-top:8px">Julio 2026 · Objetivo: visibilidad, seguidores de calidad y conversaciones por WhatsApp</p>
</div>

<div class="summary">
  <strong>Resumen:</strong> no optimizar solo a “visitas al perfil”.
  Encadenar <strong>descubrimiento → valor → seguir → consulta</strong>.
  Los anuncios aceleran el alcance; el contenido sostiene el crecimiento.
</div>

<h2>1. Estrategia orgánica (base)</h2>
<p>Frecuencia alineada a la grilla de contenido:</p>
<table>
  <thead>
    <tr><th>Canal</th><th>Ritmo</th><th>Enfoque</th></tr>
  </thead>
  <tbody>
    <tr><td>Feed</td><td>Lunes · Miércoles · Viernes</td><td>Educativo, casos, servicios / CTA</td></tr>
    <tr><td>Stories</td><td>Casi diario (1–3)</td><td>Encuestas, tips, behind the scenes</td></tr>
    <tr><td>Reels</td><td>1–2 por semana</td><td>Formato que más crece: tips + proyectos reales</td></tr>
  </tbody>
</table>
<p>Lo que más sube seguidores: <strong>Reels educativos + casos reales</strong>, no solo posts estáticos.</p>

<h2>2. Publicidad: objetivos de Meta Ads</h2>
<table>
  <thead>
    <tr><th>Objetivo</th><th>Para qué sirve</th><th>¿Para seguidores?</th></tr>
  </thead>
  <tbody>
    <tr><td>Visitas al perfil</td><td>Notoriedad rápida</td><td>Débil: mucha gente entra y no sigue</td></tr>
    <tr><td>Interacción</td><td>Likes, guardados, comentarios</td><td>Mejor que visitas</td></tr>
    <tr><td>Reproducciones / ThruPlay</td><td>Alcance barato con Reels</td><td>Bueno para descubrirte</td></tr>
    <tr><td>Mensajes (WhatsApp)</td><td>Clientes y consultas</td><td>El más valioso para el negocio</td></tr>
    <tr><td>Tráfico al sitio (/contacto)</td><td>Leads al formulario</td><td>Útil con el form activo</td></tr>
  </tbody>
</table>
<p><strong>Recomendación:</strong> no usar solo “visitas al perfil” como estrategia principal.</p>

<h2>3. Estructura de campañas (Mes 1)</h2>
<div class="phases">
  <div class="phase">
    <div class="pct">60%</div>
    <h3>A · Descubrimiento</h3>
    <p>Objetivo: reproducciones de video o interacción.</p>
    <p>Creativo: Reel (web / 5 señales / caso).</p>
    <p>Meta: que te conozcan y vean valor.</p>
  </div>
  <div class="phase">
    <div class="pct">20%</div>
    <h3>B · Perfil</h3>
    <p>Objetivo: visitas al perfil o interacción al mejor post.</p>
    <p>Solo con el Reel/carrusel que ya rindió en orgánico.</p>
    <p>Presupuesto bajo: refuerzo, no pilar.</p>
  </div>
  <div class="phase">
    <div class="pct">20%</div>
    <h3>C · Conversión</h3>
    <p>Objetivo: mensajes → WhatsApp.</p>
    <p>Copy: app o sistema para tu negocio.</p>
    <p>Aquí ganas clientes, no solo followers.</p>
  </div>
</div>

<h2>4. Presupuesto orientativo</h2>
<table>
  <thead>
    <tr><th>Nivel</th><th>Diario</th><th>Semanal</th></tr>
  </thead>
  <tbody>
    <tr><td>Test</td><td>$3–5 USD</td><td>~$25–35</td></tr>
    <tr><td>Activo</td><td>$8–15 USD</td><td>~$60–100</td></tr>
  </tbody>
</table>
<p>Priorizar <strong>1–2 creativos</strong> (Reel + carrusel de señales), no muchos anuncios a la vez.</p>

<h2>5. Creativos listos para potenciar</h2>
<ul>
  <li>Carrusel “5 señales de que necesitas un sistema empresarial”</li>
  <li>Reel scroll reveal de informaticagonzalez.com</li>
  <li>Casos: Soluciones CGT, LogiTrack, servicios</li>
</ul>
<p>Ángulo de perfil más emprendedor: <em>apps + software</em> para negocios que quieren crecer.</p>

<h2>6. KPIs a medir (Mes 1)</h2>
<table>
  <thead>
    <tr><th>Métrica</th><th>Meta sugerida</th></tr>
  </thead>
  <tbody>
    <tr><td>Publicaciones feed</td><td>12</td></tr>
    <tr><td>Reels</td><td>4–6</td></tr>
    <tr><td>Seguidores (orgánico + ads)</td><td>+80–150</td></tr>
    <tr><td>Clics link en bio</td><td>20+</td></tr>
    <tr><td>Conversaciones WhatsApp desde IG</td><td>3–5</td></tr>
    <tr><td>Guardados en posts educativos</td><td>Idealmente &gt; likes</td></tr>
  </tbody>
</table>

<h3>En Ads Manager, priorizar:</h3>
<ol>
  <li>Costo por visita al perfil</li>
  <li>% que sigue después de visitar (si Meta lo muestra)</li>
  <li>Guardados y compartidos</li>
  <li>Conversaciones de WhatsApp</li>
  <li>Clics al link en bio / formulario /contacto</li>
</ol>

<h2>7. Conclusión operativa</h2>
<div class="box">
  <p><strong>1.</strong> Mantener la grilla orgánica (Lun / Mié / Vie + Stories + Reels).</p>
  <p><strong>2.</strong> Ads: primeras campañas a <strong>reproducciones / interacción</strong> con Reels.</p>
  <p><strong>3.</strong> Usar “visitas al perfil” solo como refuerzo del mejor creativo.</p>
  <p><strong>4.</strong> Reservar parte del presupuesto a <strong>WhatsApp</strong> (clientes reales).</p>
  <p><strong>5.</strong> Medir conversaciones y guardados, no solo seguidores.</p>
</div>

<div class="footer">
  <span>Informática González · informaticagonzalez.com</span>
  <span>Instagram @informatica.gonzalez</span>
</div>

</body>
</html>`;

const browser = await chromium.launch();
const page = await browser.newPage();
await page.setContent(html, { waitUntil: 'networkidle' });
await page.pdf({
  path: outPath,
  format: 'A4',
  printBackground: true,
  margin: { top: '12mm', bottom: '12mm', left: '12mm', right: '12mm' },
});
await browser.close();

console.log(`PDF generado: ${outPath}`);

File: scripts/generate-plan-crecimiento-pdf.py
"""Genera plan-crecimiento-instagram.pdf — Informática González."""
from pathlib import Path

from reportlab.lib import colors
from reportlab.lib.enums import TA_LEFT
from reportlab.lib.pagesizes import A4
from reportlab.lib.styles import ParagraphStyle, getSampleStyleSheet
from reportlab.lib.units import mm
from reportlab.platypus import (
    Paragraph,
    SimpleDocTemplate,
    Spacer,
    Table,
    TableStyle,
    KeepTogether,
    HRFlowable,
)

OUT = Path(__file__).resolve().parents[1] / "marketing" / "plan-crecimiento-instagram.pdf"

BLUE = colors.HexColor("#2563eb")
DARK = colors.HexColor("#0a0a0a")
GRAY = colors.HexColor("#525252")
LIGHT = colors.HexColor("#f5f5f5")
BORDER = colors.HexColor("#e5e5e5")
BG_TIP = colors.HexColor("#eff6ff")


def styles():
    base = getSampleStyleSheet()
    return {
        "hero_title": ParagraphStyle(
            "hero_title",
            parent=base["Heading1"],
            fontName="Helvetica-Bold",
            fontSize=20,
            textColor=colors.white,
            spaceAfter=4,
            leading=24,
        ),
        "hero_sub": ParagraphStyle(
            "hero_sub",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=10,
            textColor=colors.HexColor("#a3a3a3"),
            leading=14,
        ),
        "h2": ParagraphStyle(
            "h2",
            parent=base["Heading2"],
            fontName="Helvetica-Bold",
            fontSize=11,
            textColor=BLUE,
            spaceBefore=14,
            spaceAfter=8,
            leading=14,
        ),
        "h3": ParagraphStyle(
            "h3",
            parent=base["Heading3"],
            fontName="Helvetica-Bold",
            fontSize=10,
            textColor=DARK,
            spaceBefore=8,
            spaceAfter=4,
        ),
        "body": ParagraphStyle(
            "body",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=9.5,
            textColor=GRAY,
            leading=13,
            spaceAfter=6,
        ),
        "bullet": ParagraphStyle(
            "bullet",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=9.5,
            textColor=GRAY,
            leading=13,
            leftIndent=12,
            spaceAfter=3,
        ),
        "cell": ParagraphStyle(
            "cell",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=8.5,
            textColor=DARK,
            leading=11,
        ),
        "cell_h": ParagraphStyle(
            "cell_h",
            parent=base["Normal"],
            fontName="Helvetica-Bold",
            fontSize=8,
            textColor=GRAY,
            leading=10,
        ),
        "tip": ParagraphStyle(
            "tip",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=9.5,
            textColor=DARK,
            leading=13,
        ),
        "footer": ParagraphStyle(
            "footer",
            parent=base["Normal"],
            fontName="Helvetica",
            fontSize=8,
            textColor=GRAY,
        ),
    }


def table(data, col_widths):
    t = Table(data, colWidths=col_widths, hAlign="LEFT")
    t.setStyle(
        TableStyle(
            [
                ("BACKGROUND", (0, 0), (-1, 0), LIGHT),
                ("TEXTCOLOR", (0, 0), (-1, 0), GRAY),
                ("FONTNAME", (0, 0), (-1, 0), "Helvetica-Bold"),
                ("FONTSIZE", (0, 0), (-1, -1), 8.5),
                ("GRID", (0, 0), (-1, -1), 0.5, BORDER),
                ("VALIGN", (0, 0), (-1, -1), "TOP"),
                ("LEFTPADDING", (0, 0), (-1, -1), 6),
                ("RIGHTPADDING", (0, 0), (-1, -1), 6),
                ("TOPPADDING", (0, 0), (-1, -1), 5),
                ("BOTTOMPADDING", (0, 0), (-1, -1), 5),
                ("BACKGROUND", (0, 1), (-1, -1), colors.white),
            ]
        )
    )
    return t


def hero_block(s):
    inner = [
        Paragraph("MARKETING · INSTAGRAM", ParagraphStyle(
            "badge", fontName="Helvetica-Bold", fontSize=8, textColor=BLUE, spaceAfter=6
        )),
        Paragraph("Plan de crecimiento de seguidores", s["hero_title"]),
        Paragraph(
            "<b>Informática González</b> · Estrategia orgánica + Meta Ads<br/>"
            "Julio 2026 · Visibilidad, seguidores de calidad y conversaciones por WhatsApp",
            s["hero_sub"],
        ),
    ]
    box = Table([[inner]], colWidths=[178 * mm])
    box.setStyle(
        TableStyle(
            [
                ("BACKGROUND", (0, 0), (-1, -1), DARK),
                ("LEFTPADDING", (0, 0), (-1, -1), 14),
                ("RIGHTPADDING", (0, 0), (-1, -1), 14),
                ("TOPPADDING", (0, 0), (-1, -1), 14),
                ("BOTTOMPADDING", (0, 0), (-1, -1), 14),
            ]
        )
    )
    return box


def tip_box(text, s):
    t = Table([[Paragraph(text, s["tip"])]], colWidths=[178 * mm])
    t.setStyle(
        TableStyle(
            [
                ("BACKGROUND", (0, 0), (-1, -1), BG_TIP),
                ("LEFTPADDING", (0, 0), (-1, -1), 10),
                ("RIGHTPADDING", (0, 0), (-1, -1), 10),
                ("TOPPADDING", (0, 0), (-1, -1), 8),
                ("BOTTOMPADDING", (0, 0), (-1, -1), 8),
                ("LINEBEFORE", (0, 0), (0, 0), 3, BLUE),
            ]
        )
    )
    return t


def phase_cards(s):
    phases = [
        (
            "60%",
            "A · Descubrimiento",
            "Reproducciones o interacción.<br/>Creativo: Reel (web / 5 señales / caso).<br/>Meta: que te conozcan.",
        ),
        (
            "20%",
            "B · Perfil",
            "Visitas al perfil o interacción al mejor post.<br/>Solo creativos que ya rindieron.<br/>Presupuesto bajo.",
        ),
        (
            "20%",
            "C · Conversión",
            "Mensajes → WhatsApp.<br/>Apps y software para negocios.<br/>Clientes, no solo followers.",
        ),
    ]
    cells = []
    for pct, title, body in phases:
        cells.append(
            [
                Paragraph(f'<font color="#2563eb" size="16"><b>{pct}</b></font>', s["body"]),
                Paragraph(f"<b>{title}</b>", s["h3"]),
                Paragraph(body, s["body"]),
            ]
        )
    data = [[c for c in cells]]
    # Actually need 3 columns each with stacked content
    col_content = []
    for pct, title, body in phases:
        inner = Table(
            [
                [Paragraph(f'<font color="#2563eb" size="14"><b>{pct}</b></font>', s["body"])],
                [Paragraph(f"<b>{title}</b>", ParagraphStyle("pt", fontName="Helvetica-Bold", fontSize=9, textColor=DARK, spaceAfter=4))],
                [Paragraph(body, ParagraphStyle("pb", fontName="Helvetica", fontSize=8, textColor=GRAY, leading=11))],
            ],
            colWidths=[54 * mm],
        )
        inner.setStyle(
            TableStyle(
                [
                    ("BOX", (0, 0), (-1, -1), 0.5, BORDER),
                    ("LEFTPADDING", (0, 0), (-1, -1), 8),
                    ("RIGHTPADDING", (0, 0), (-1, -1), 8),
                    ("TOPPADDING", (0, 0), (-1, -1), 8),
                    ("BOTTOMPADDING", (0, 0), (-1, -1), 8),
                    ("VALIGN", (0, 0), (-1, -1), "TOP"),
                ]
            )
        )
        col_content.append(inner)

    wrap = Table([col_content], colWidths=[58 * mm, 58 * mm, 58 * mm])
    wrap.setStyle(TableStyle([("LEFTPADDING", (0, 0), (-1, -1), 0), ("RIGHTPADDING", (0, 0), (-1, -1), 4), ("VALIGN", (0, 0), (-1, -1), "TOP")]))
    return wrap


def build():
    s = styles()
    doc = SimpleDocTemplate(
        str(OUT),
        pagesize=A4,
        leftMargin=16 * mm,
        rightMargin=16 * mm,
        topMargin=14 * mm,
        bottomMargin=14 * mm,
        title="Plan de crecimiento Instagram — Informática González",
        author="Informática González",
    )

    c = s["cell"]
    ch = s["cell_h"]
    story = []

    story.append(hero_block(s))
    story.append(Spacer(1, 10))
    story.append(
        tip_box(
            "<b>Resumen:</b> no optimizar solo a “visitas al perfil”. "
            "Encadenar <b>descubrimiento → valor → seguir → consulta</b>. "
            "Los anuncios aceleran el alcance; el contenido sostiene el crecimiento.",
            s,
        )
    )

    story.append(Paragraph("1. Estrategia orgánica (base)", s["h2"]))
    story.append(Paragraph("Frecuencia alineada a la grilla de contenido:", s["body"]))
    story.append(
        table(
            [
                [Paragraph("Canal", ch), Paragraph("Ritmo", ch), Paragraph("Enfoque", ch)],
                [Paragraph("Feed", c), Paragraph("Lunes · Miércoles · Viernes", c), Paragraph("Educativo, casos, servicios / CTA", c)],
                [Paragraph("Stories", c), Paragraph("Casi diario (1–3)", c), Paragraph("Encuestas, tips, behind the scenes", c)],
                [Paragraph("Reels", c), Paragraph("1–2 por semana", c), Paragraph("Tips + proyectos reales (más crecimiento)", c)],
            ],
            [35 * mm, 55 * mm, 88 * mm],
        )
    )
    story.append(
        Paragraph(
            "Lo que más sube seguidores: <b>Reels educativos + casos reales</b>, no solo posts estáticos.",
            s["body"],
        )
    )

    story.append(Paragraph("2. Publicidad: objetivos de Meta Ads", s["h2"]))
    story.append(
        table(
            [
                [Paragraph("Objetivo", ch), Paragraph("Para qué sirve", ch), Paragraph("¿Para seguidores?", ch)],
                [Paragraph("Visitas al perfil", c), Paragraph("Notoriedad rápida", c), Paragraph("Débil: muchos entran y no siguen", c)],
                [Paragraph("Interacción", c), Paragraph("Likes, guardados, comentarios", c), Paragraph("Mejor que visitas", c)],
                [Paragraph("Reproducciones / ThruPlay", c), Paragraph("Alcance barato con Reels", c), Paragraph("Bueno para descubrirte", c)],
                [Paragraph("Mensajes (WhatsApp)", c), Paragraph("Clientes y consultas", c), Paragraph("El más valioso para el negocio", c)],
                [Paragraph("Tráfico al sitio (/contacto)", c), Paragraph("Leads al formulario", c), Paragraph("Útil con el form activo", c)],
            ],
            [48 * mm, 65 * mm, 65 * mm],
        )
    )
    story.append(
        Paragraph(
            "<b>Recomendación:</b> no usar solo “visitas al perfil” como estrategia principal.",
            s["body"],
        )
    )

    story.append(Paragraph("3. Estructura de campañas (Mes 1)", s["h2"]))
    story.append(phase_cards(s))
    story.append(Spacer(1, 6))

    story.append(Paragraph("4. Presupuesto orientativo", s["h2"]))
    story.append(
        table(
            [
                [Paragraph("Nivel", ch), Paragraph("Diario", ch), Paragraph("Semanal", ch)],
                [Paragraph("Test", c), Paragraph("$3–5 USD", c), Paragraph("~$25–35", c)],
                [Paragraph("Activo", c), Paragraph("$8–15 USD", c), Paragraph("~$60–100", c)],
            ],
            [50 * mm, 64 * mm, 64 * mm],
        )
    )
    story.append(
        Paragraph(
            "Priorizar <b>1–2 creativos</b> (Reel + carrusel de señales), no muchos anuncios a la vez.",
            s["body"],
        )
    )

    story.append(Paragraph("5. Creativos listos para potenciar", s["h2"]))
    for item in [
        "• Carrusel “5 señales de que necesitas un sistema empresarial”",
        "• Reel scroll reveal de informaticagonzalez.com",
        "• Casos: Soluciones CGT, LogiTrack, servicios",
        "• Ángulo emprendedor juvenil: apps + software para negocios que quieren crecer",
    ]:
        story.append(Paragraph(item, s["bullet"]))

    story.append(Paragraph("6. KPIs a medir (Mes 1)", s["h2"]))
    story.append(
        table(
            [
                [Paragraph("Métrica", ch), Paragraph("Meta sugerida", ch)],
                [Paragraph("Publicaciones feed", c), Paragraph("12", c)],
                [Paragraph("Reels", c), Paragraph("4–6", c)],
                [Paragraph("Seguidores (orgánico + ads)", c), Paragraph("+80–150", c)],
                [Paragraph("Clics link en bio", c), Paragraph("20+", c)],
                [Paragraph("Conversaciones WhatsApp desde IG", c), Paragraph("3–5", c)],
                [Paragraph("Guardados en posts educativos", c), Paragraph("Idealmente > likes", c)],
            ],
            [110 * mm, 68 * mm],
        )
    )
    story.append(Paragraph("En Ads Manager, priorizar:", s["h3"]))
    for i, item in enumerate(
        [
            "Costo por visita al perfil",
            "% que sigue después de visitar (si Meta lo muestra)",
            "Guardados y compartidos",
            "Conversaciones de WhatsApp",
            "Clics al link en bio / formulario /contacto",
        ],
        1,
    ):
        story.append(Paragraph(f"{i}. {item}", s["bullet"]))

    story.append(Paragraph("7. Conclusión operativa", s["h2"]))
    story.append(
        tip_box(
            "<b>1.</b> Mantener la grilla orgánica (Lun / Mié / Vie + Stories + Reels).<br/>"
            "<b>2.</b> Ads: primeras campañas a <b>reproducciones / interacción</b> con Reels.<br/>"
            "<b>3.</b> Usar “visitas al perfil” solo como refuerzo del mejor creativo.<br/>"
            "<b>4.</b> Reservar parte del presupuesto a <b>WhatsApp</b> (clientes reales).<br/>"
            "<b>5.</b> Medir conversaciones y guardados, no solo seguidores.",
            s,
        )
    )

    story.append(Spacer(1, 16))
    story.append(HRFlowable(width="100%", thickness=0.5, color=BORDER))
    story.append(Spacer(1, 6))
    story.append(
        Paragraph(
            "Informática González · informaticagonzalez.com · Instagram @informatica.gonzalez",
            s["footer"],
        )
    )

    doc.build(story)
    print(f"PDF generado: {OUT}")


if __name__ == "__main__":
    build()
